cpp/include/cudf/lists/contains.hpp

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2020, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <cudf/column/column.hpp>
+#include <cudf/lists/lists_column_view.hpp>
+
+namespace cudf {
+namespace lists {
+
```

Comment:
New lists API should be added to the `lists`_apis doxygen group.
For example: https://github.com/rapidsai/cudf/blob/02e25b6f3db4616183a5e0b590ae01c510414d69/cpp/include/cudf/lists/extract.hpp#L24-L28
Don't forget the end group: https://github.com/rapidsai/cudf/blob/02e25b6f3db4616183a5e0b590ae01c510414d69/cpp/include/cudf/lists/extract.hpp#L68

You can call it `lists_contains` and then also add it to the `doxygen_groups.h` here: https://github.com/rapidsai/cudf/blob/02e25b6f3db4616183a5e0b590ae01c510414d69/cpp/include/doxygen_groups.h#L143-L146

Looks like `lists_gather` group is missing from this section of `doxygen_groups.h` as well.
https://github.com/rapidsai/cudf/blob/02e25b6f3db4616183a5e0b590ae01c510414d69/cpp/include/cudf/lists/gather.hpp#L25

Reply:
Thank you for pointing this out, @davidwendt. I hadn't wrapped my head around how to use Doxygen's directives. I'll incorporate your suggestions now.